### PR TITLE
New Crochet Pattern Cheat Sheet

### PR DESCRIPTION
**Pull request by b2ddg**
**What does your Instant Answer do?**
 My instant answer provides abbreviations for crochet patterns

**What problem does your Instant Answer solve (Why is it better than organic links)?**
 Crochet patterns use a code that is referenced often when learning

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://craftyarncouncil.com/crochet.html

**Why did you choose this data source?**
 It was recommended to me as canonical

**Are there any other alternative (better) data sources?**
 I believe most sources are based on this one.

**What are some example queries that trigger this Instant Answer?**
 'crochet pattern cheat sheet' 

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
crocheters, crafters

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
no

**Which existing Instant Answers will this one supercede/overlap with?**
none

**Are you having any problems? Do you need our help with anything?**
nope

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-10-12 at 9 37 59 pm

## Checklist
Please place an 'X' where appropriate.

```
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:

    - Mac OSX
        - [] Google Chrome
        - [x] Firefox
        - [] Opera
        - [x] Safari

    - iOS (iPhone)
        - [x] Safari Mobile
        - [] Google Chrome
        - [] Opera

```

-----
IA Page: https://duck.co/ia/view/crochet_pattern_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/crochet-pattern.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            "val": "contrasting color",
+            "key": "CC"
+        }, {
+            "val": "chain stitch",
+            "key": "ch"
+        }, {
+            "val": "refers to chain or space previously made: e.g., ch-1 space",
+            "key": "ch-"
+        }],
+        " ": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/crochet-pattern.json]: I see what you did here...
clever-girl

Comment (b2ddg) [inline share/goodie/cheat_sheets/json/crochet-pattern.json]: hahaha, what's the correct way to do this??

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

**Comment by moollaza** [comment]
This LGTM :+1:

**Comment by b2ddg** [comment]
woot woot

**Comment by moollaza** [comment]
@b2ddg congrats! It's live: https://duckduckgo.com/?q=Crochet%20Pattern%20Cheat%20Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**